environment.yml

```diff
@@ -17,4 +18,4 @@ dependencies:
     - pytest
     - pytest-cov
     - git+https://bitbucket.org/cmelab/cme_utils.git
-    - git+https://github.com/cmelab/planckton@test-docker
+    - git+https://github.com/cmelab/planckton@fix_setup
```

Comment:
Will this stay on `fix_setup` post merge? If this was for CI, we should always be testing on the code in the PR. Or is this just and edge case and once this gets fixed we don't need this hack?

Reply:
good point, fix_setup is a tag, not this branch. I'm not sure how to handle updating planckton in the environment and in the image. should the environment always install the current master?

Reply:
Ah the `@` notation works for tags and branches so it can get confusing! Unless we build the docker image we use to test as part of the CI, we will always have some weirdness when we want to add something to the docker image as part of a PR. I'm not sure the best way to handle this.

Reply:
I think in a perfect world, we build the docker image as part of our CI and test with that image. We should have some caching so that if the docker file and the `environment.yml` haven't changed, then we just use the docker image we have on the hub, if the files have changed, then we build a new image. Then after the PR is merged, that new image should be pushed to the hub. 

I think then we might want to switch to using the "latest" image when testing, and then for versioning docker images, we should create separate PRs where we also tag planckton releases. I'll figure out how to do this and in the mean time we can do what are are doing to manually build docker images when we need to test new dependencies.

Reply:
OK, awesome. I'm going to merge this PR, but I will make an issue to setup caching--could we work on that together?